### PR TITLE
Docs to describe networking options.

### PR DESCRIPTION
**Pull request by simonwo**

### CODE REVIEW

**Comment by fakela** [comment]
@simonwo I have reviewed the doc, before merging, please take a final look at it and see if the changes I made are okay

**Review by fakela — APPROVED** [review]

File: docs/next-steps/networking.md [hunk 1]
```diff
@@ -3,32 +3,32 @@ sidebar_position: 4
...
 
-Jobs can write results back to a temporary IPFS node run by Bacalhau, to a persistent IPFS storage location, or directly to a Filecoin storage provider. By default, jobs will write results to both a Bacalhau IPFS node and persistently to Filecoin via [Estuary](https://estuary.tech). See [the documentation](../all-flags.md) on the `--publisher` command line flag for how to configure this.
+You can write back results from your Bacalhau jobs to; a tempoary Bacalhau IPFS node,  persistent IPFS storage location, or directly to a Filecoin storage provider. By default, jobs will write results to both a Bacalhau IPFS node and persistently to Filecoin through [Estuary](https://estuary.tech) using the `--publisher` command line flag for . See [command line flags](https://docs.bacalhau.org/all-flags) on how to configure this.
```
Comment (simonwo) [inline docs/next-steps/networking.md:18]: Sentence here seems to end abruptly?

**Review by simonwo — COMMENTED** [review]
LGTM!

**Review by enricorotundo — APPROVED** [review]

File: docs/next-steps/networking.md (hunk 1 above)
Comment (fakela) [reply docs/next-steps/networking.md:18]: @simonwo, is it this line you are refering to: . _See [command line flags](https://docs.bacalhau.org/all-flags) on how to configure this._

Comment (simonwo) [reply docs/next-steps/networking.md:18]: No sorry, 
> using the `--publisher` command line flag for .

Comment (fakela) [reply docs/next-steps/networking.md:18]: oh, thank you for catching that

**Pull request merged**